assets/stylesheets/_components.scss

```diff
@@ -143,6 +142,7 @@
 @import 'components/web-preview/style';
 @import 'blocks/credit-card-form/style';
 @import 'devdocs/style';
+@import 'devdocs/design/docs-example/style';
```

Comment:
I think we could place the docs-example component directly in devdocs (not under design).